Common/Extensions/NSBundle.swift

```diff
@@ -85,6 +85,19 @@ extension Bundle {
         }
     }
 
+    // These strings are only configured if it is a workspace build
+    var workspaceBuild: String? {
+        return object(forInfoDictionaryKey: "com-loopkit-LoopWorkspace-y-n") as? String
```

Comment:
I'm not sure what the y-n means? If it's to communicate whether or not is a workspace build, it should be something like com-loopkit-isWorkspaceBuild and be a boolean.  Also, it may be unneeded if the following are only set if it's a workspace build, as we can tell by their presence.
All three var names should probably share the same naming scheme.
Prefer full names to abbreviations: e.g. workspaceGitRevision vs gitRevWs

Reply:
I kind of fumbled around trying to figure out how to do this.
I don't know how to do the boolean, so I took it out.
Redid the names and tested locally.